src/utils/DockerUtil.js

```diff
@@ -274,6 +311,15 @@ export default {
     });
   },
 
+  status (name) {
+    let running = false;
+    let container = this.client.getContainer(name);
```

Comment:
This method was added to check the status of the container linked, preventing non-running (or incorrect) containers from being used for linking.